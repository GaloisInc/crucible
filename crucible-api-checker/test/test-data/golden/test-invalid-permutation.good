[Crux] Simulating function "main"
[Crux] Failed to prove 1 out of 1 goals.
[Crux] Counter example for function dependency error at: _Z1Cvinternal
in _Z1Cv at internal
