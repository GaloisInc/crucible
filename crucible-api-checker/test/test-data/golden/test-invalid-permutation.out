[Crux] `clang` compilation failed.
*** Exit code: 1
*** Standard out:
*** Standard error:
   llvm-link: c-src/libcxx-8.0.0.bc: error: Could not open input file: No such file or directory
   llvm-link: error:  loading file 'c-src/libcxx-8.0.0.bc'

